packages/components/tests/dummy/app/router.js

```diff
@@ -42,6 +42,7 @@ Router.map(function () {
       this.route('textarea');
       this.route('toggle');
       this.route('radio-card');
+      this.route('file-input');
```

Comment:
Would be good if we can sort these alphabetically – it was likely me who placed the `radio-card` at the end 🤦